### PR TITLE
decapitate pdf, fixed laziness with logging

### PR DESCRIPTION
**Pull request by lsh-0**

### CODE REVIEW

File: packagepoa/transform.py [hunk 1]
```diff
@@ -222,10 +221,10 @@ def copy_pdf_to_output_dir(
             if file_content:
                 with open(os.path.join(output_dir, new_name), "wb") as out_handler:
                     out_handler.write(file_content)
-        except IOError:
+        except IOError as ioe:
             # The decap may return true but the file does not exist for some reason
             #  allow the transformation to continue in order to processes the supplementary files
-            LOGGER.error("decap returned true but the pdf file is missing %s", new_name)
+            LOGGER.error("decap returned true but the pdf file is missing %s: %s", new_name, ioe)
```
Comment (lsh-0) [inline packagepoa/transform.py:227]: this might give us some more insight in to the actual error.

Comment (gnott) [reply packagepoa/transform.py:227]: Agreed, I want to know more!

**Review by gnott — APPROVED** [review]
More logging, and tests are green, sounds good.

If you want to merge this PR, please be welcome to release a new library version too, but if you want me to generate the release and deploy it I can.

**Comment by lsh-0** [comment]
kicked this https://alfred.elifesciences.org/job/release/job/release-package-poa/
which triggered this https://alfred.elifesciences.org/job/dependencies/job/dependencies-elife-bot-update-dependency/
that created this https://github.com/elifesciences/elife-bot/pull/1623

**Pull request merged**